### DISCUSSION TITLE
how-to in adding server side mods

### DISCUSSION THREAD

**Discussion by froghilda** [mbround18/valheim-docker#310] [Q&A] [answered]
I have the code base for all the mods I want to add to x server. How do I do it? I run an AWS lights ail instance with a container of your image.

**Comment by mbround18** [comment, marked as answer]
Very simple, there is a TYPE variable you can change and a MODS variable you can add. By changing the type, you can than set what kind of server you want to run and by adding the MODS variable the container will download the mods for you. 

Here is a full guide: [Getting Started with Mods](https://github.com/mbround18/valheim-docker/blob/main/docs/getting_started_with_mods.md)
